src/main/scala/ie/zalando/fabric/gateway/web/GatewayWebhookRoutes.scala

```diff
@@ -25,23 +25,25 @@ trait GatewayWebhookRoutes extends JsonModels {
       pathEnd {
         post {
           trace("metacontroller_synchronization") { span =>
-            entity(as[SynchRequest]) { synchRequest =>
-              extractExecutionContext { ec =>
-                implicit val execCtxt: ExecutionContext = ec
-                complete {
-                  ingressTransitions
-                    .defineSafeRouteTransition(synchRequest.controlledResource.spec,
-                                               synchRequest.controlledResource.metadata,
-                                               synchRequest.currentState.values.toSeq)
-                    .map { ingressDefinitions =>
-                      val response = SynchResponse(
-                        GatewayStatus(synchRequest.currentState.size, synchRequest.currentState.keySet),
-                        ingressDefinitions
-                      )
-                      log.debug(s"Synch Request response: $response")
-                      span.setTag("response_payload", response.toString)
-                      response
-                    }
+            withoutSizeLimit {
```

Comment:
This was added due to an issue I hit when we were processing very large request payloads. It's not currently happening after several rounds of testing but might be best to leave it in seeing as the request payload size can depend on number of managed resources. Basically Akka HTTP won't process very large payload sizes by default as DDOS prevention tactic.